tests/installation/installation_overview.pm

```diff
@@ -26,6 +26,13 @@ sub run() {
     # preserve it for the video
     wait_idle 10;
 
+    # Check autoyast has been removed in SP2
+    if (get_var("SP2ORLATER")) {
+        send_key 'tab';
+        send_key 'pgdn';
```

Comment:
hm, what happens e.g. in an *allpatterns* scenario when the installation overview covers more than the one-and-half pages that you would reach with one press of "pgdn"? Take a look here: https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/installation/installation_overview.pm#L58
Wouldn't this also make sense here?